src/EventStore.ClientAPI/IEventStoreConnection.cs

```diff
@@ -49,111 +50,114 @@ public interface IEventStoreConnection : IDisposable
         Task<DeleteResult> DeleteStreamAsync(string stream, long expectedVersion, UserCredentials userCredentials = null);
 
         /// <summary>
-        /// Deletes a stream from the Event Store asynchronously
+        /// Deletes a stream from Event Store asynchronously.
         /// </summary>
         /// <param name="stream">The name of the stream to delete.</param>
         /// <param name="expectedVersion">The expected version that the streams should have when being deleted. <see cref="ExpectedVersion"/></param>
         /// <param name="hardDelete">Indicator for tombstoning vs soft-deleting the stream. Tombstoned streams can never be recreated. Soft-deleted streams
+        /// TODO: Can the below be a link?
         /// can be written to again, but the EventNumber sequence will not start from 0.</param>
         /// <param name="userCredentials">The optional user credentials to perform operation with.</param>
         /// <returns>A <see cref="Task&lt;DeleteResult&gt;"/> containing the results of the delete stream operation.</returns>
         Task<DeleteResult> DeleteStreamAsync(string stream, long expectedVersion, bool hardDelete, UserCredentials userCredentials = null);
 
+        /// TODO: Differences?
```

Comment:
The three `AppendToStreamAsync` methods are essentially the same, they just have different signatures.